### DISCUSSION TITLE
can not get correct remote address

### DISCUSSION THREAD

**Discussion by cjw0202** [fastapi/fastapi#8731] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
## main.py

from fastapi import FastAPI

app = FastAPI()

@app.get('/')
async def root():
    return {'code': 200, 'msg': 'Hello, FastApi'}
```

### Description

# C:\users\cjw\\.virtualenvs\fastapi-v0vfbmzh\lib\site-packages\uvicorn\protocols\utils.py        print(socket_info)
```
fastapi-v0VFBMzH) (fastapi) D:\Workspace\fastapi>uvicorn main:app --reload
INFO:     Will watch for changes in these directories: ['D:\\Workspace\\fastapi']
INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [15856] using statreload
INFO:     Started server process [3544]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
<socket.socket fd=780, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 8000), raddr=(0, b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00')>  
Fatal error in server handler
protocol: <uvicorn.protocols.http.h11_impl.H11Protocol object at 0x000001C564F6BF88>
transport: <_SelectorSocketTransport fd=780 read=polling write=<idle, bufsize=0>>
Traceback (most recent call last):
  File "c:\users\cjw\\.virtualenvs\fastapi-v0vfbmzh\lib\site-packages\uvicorn\server.py", line 108, in handler
    reader, writer, server_state=self.server_state, config=config
  File "c:\users\cjw\\.virtualenvs\fastapi-v0vfbmzh\lib\site-packages\uvicorn\_handlers\http.py", line 75, in handle_http
    protocol.connection_made(transport)
  File "c:\users\cjw\\.virtualenvs\fastapi-v0vfbmzh\lib\site-packages\uvicorn\protocols\http\h11_impl.py", line 85, in connection_made
    self.client = get_remote_addr(transport)
  File "c:\users\cjw\\.virtualenvs\fastapi-v0vfbmzh\lib\site-packages\uvicorn\protocols\utils.py", line 15, in get_remote_addr
    return (str(info[0]), int(info[1])) if isinstance(info, tuple) else None
ValueError: invalid literal for int() with base 10: b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
```

the raddr is (0, b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00')，something wrong ？

### Operating System

Windows

### Operating System Details

windows 10

### FastAPI Version

0.70.1

### Python Version

Python 3.7.9

**Comment by n8sty** [comment]
Looks like you might be having a dependency error with h11 - try installing `uvicorn[standard]` (https://www.uvicorn.org/#quickstart)
